backend/src/main/java/gov/cdc/usds/simplereport/service/errors/ExperianPersonMatchException.java

```diff
@@ -0,0 +1,17 @@
+package gov.cdc.usds.simplereport.service.errors;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * An error indicating Experian's Knowledge IQ could not match a person in order to provide
+ * questions to be answered for identity verification. {@link HttpStatus#BAD_REQUEST}.
+ */
+@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Did we settle on BAD_REQUEST or NOT_FOUND here? These two seem to be conflicting.

Reply:
Doh, had also made that change in the other branch.